B 22097 int

loganwc opened the pull request:
## [Agility ticket](tbd)

## Summary

Made the radio buttons horizontal instead of vertical 

### How to test

1. Login as a customer
2. Create a new move
3. Create a ppm
4. 2nd/3rd address toggle is horizontal instead of vertical 

## Screenshots
![Screenshot 2025-03-03 at 12 25 19 PM](https://github.com/user-attachments/assets/d9bbe23c-80be-4b09-a383-033e5ee0a728)

ajlusk reviewed (approved):
<img width="349" alt="Screenshot 2025-03-04 at 10 09 50 AM" src="https://github.com/user-attachments/assets/fca6cc16-c2fe-4ed5-b033-976795c73d54" />

Looks good. Meets AC.

Backend changes are only from the linter. I'm seeing the same thing on one of my PRs.

pambecker commented:
The BL also says to address the size difference in the 2nd/3rd address headers.  I'm seeing a difference in the Customer HHG side of 2nd/3rd Pickup headers.
And then this also brings up the fact that the 'Second Pickup Address' is above the radio buttons on HHG.  but 'Third Pickup Address' header is below those radio buttons. (same for Delivery, 2nd is above and 3rd is below radio buttons). I think we should be consistent here too.  
NOTE:  PPM has all the Addr headers below the radio buttons, so that might be the way we want to go.

![image](https://github.com/user-attachments/assets/2164d36c-c630-4054-bf12-5cb1fd780404)

loganwc commented:
> The BL also says to address the size difference in the 2nd/3rd address headers. I'm seeing a difference in the Customer HHG side of 2nd/3rd Pickup headers. And then this also brings up the fact that the 'Second Pickup Address' is above the radio buttons on HHG. but 'Third Pickup Address' header is below those radio buttons. (same for Delivery, 2nd is above and 3rd is below radio buttons). I think we should be consistent here too. NOTE: PPM has all the Addr headers below the radio buttons, so that might be the way we want to go.
>

Moved the labels under the radio buttons and made them all the same size

pambecker commented:
Changes look good now, just some failing client tests based on moving that header down
MtoShipmentForm.test.jsx

loganwc commented:
> Changes look good now, just some failing client tests based on moving that header down MtoShipmentForm.test.jsx

updated tests, I removed the check looking for the label because that label is no longer on the page for those test situations

pambecker commented:
still seeing a failing client test
MtoShipmentForm.test.jsx:1907

loganwc commented:
> still seeing a failing client test MtoShipmentForm.test.jsx:1907

![Screenshot 2025-03-06 at 12 03 20 PM](https://github.com/user-attachments/assets/fbb552e1-db87-4303-89d2-2a7b72d660b8)
everything is passing on my int and story branch

paulstonebraker commented:
Just got latest and mtoshipmentsform seems to pass all tests
![image](https://github.com/user-attachments/assets/afc139d8-febb-49e5-a6e7-a10d30edef28)

pambecker commented:
> Just got latest and mtoshipmentsform seems to pass all tests ![image](https://private-user-images.githubusercontent.com/147535220/420052798-afc139d8-febb-49e5-a6e7-a10d30edef28.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDEyOTAxMTMsIm5iZiI6MTc0MTI4OTgxMywicGF0aCI6Ii8xNDc1MzUyMjAvNDIwMDUyNzk4LWFmYzEzOWQ4LWZlYmItNDllNS1hNmU3LWExMGQzMGVkZWYyOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMzA2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDMwNlQxOTM2NTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kMGIzOTU0Nzc0NWZlNTg2NjMxM2Y5M2RhMWM4ZWM0ZTViOGYzYzM4N2ExNzI5ZDExNTk1YmE2ZGVmNDMyODk4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.NVz6QfB0XeBYJmDa1m8Y3L9YiPiNlBphfqrixAyxPl8)

ok, all are passing now, for some reason it wasn't picking up your latest changes until I switched branches and came back.

pambecker commented on src/components/Customer/MtoShipmentForm/MtoShipmentForm.jsx:
```diff
@@ -451,7 +453,7 @@ class MtoShipmentForm extends Component {
                                   hasSecondaryPickup === 'true' && (
                                     <>
-                                      <h3>Third Pickup Address</h3>
+                                      <h4>Third Pickup Address</h4>
```
okay, since you are making these h4 (which I actually like better), I think we should have the PPM also show them as h4.  Since this BL is kind of comparing the 2 and a asking that they be consistent

loganwc replied:
good call, updated the ppm headers to match the hhg

pambecker commented:
The PPM 2nd/3rd Address headers are no longer bold.  (HHG are still good)

PPM:
![image](https://github.com/user-attachments/assets/36fd886c-a75b-459d-baac-8e88ad78df0c)

HHG:
![image](https://github.com/user-attachments/assets/9e9cd1b0-3f66-4a59-b7fa-c187efde4977)

antgmann reviewed (commented):
🦛 ✅ , going to do a quick review for ya
